go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/tetratelabs/proxy-wasm-go-sdk
 
 go 1.16
 
-require github.com/stretchr/testify v1.7.0
+require (
+	fortio.org/fortio v1.17.0
```

Comment:
I don't want force SDK users to add fortio (E2E test in this SDK's dep) as their dependency tree. So I would suggest to have another go.sum/mod in e2e/ directory and add fortio as a dependency there. This way no user would see fortio in their go.mod.

Reply:
done